src/components/BookList/BookList.js

```diff
@@ -1,17 +1,13 @@
-import React from 'react';
-import BookItem from '../BookItem/BookItem';
+import BookItem from "../BookItem/BookItem"
 
 const BookList = ({ bookList }) => {
     return (
-        <ul className='cardsList'>
-            {bookList.map((item) => 
-                <BookItem 
-                    key={item.title}
-                    item={item} 
-                />
-            )}
-        </ul>
-    );
+        <section>
+            {bookList.map((item) => (
+                <BookItem item={item} />
+            ))}
+        </section>
+    )
```

Comment:
Can you revert these changes before merging?

Reply:
Just did!